From e11f6576c1e747ae56911130137593908583dd1c Mon Sep 17 00:00:00 2001
From: Insu Yun <wuninsu@gmail.com>
Date: Thu, 28 Jan 2016 19:04:28 -0500
Subject: [PATCH] gma500: handling failed allocation

Since drm_property_create_range can be failed in memory pressure, 
it needs to be handled.

Signed-off-by: Insu Yun <wuninsu@gmail.com>
---
 drivers/gpu/drm/gma500/framebuffer.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/gma500/framebuffer.c b/drivers/gpu/drm/gma500/framebuffer.c
index cb95765..31085e4 100644
--- a/drivers/gpu/drm/gma500/framebuffer.c
+++ b/drivers/gpu/drm/gma500/framebuffer.c
@@ -683,6 +683,8 @@ static int psb_create_backlight_property(struct drm_device *dev)
 		return 0;
 
 	backlight = drm_property_create_range(dev, 0, "backlight", 0, 100);
+	if (!backlight)
+		return -ENOMEM;
 
 	dev_priv->backlight_property = backlight;
 
-- 
1.9.1

